core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.directive.html

```diff
@@ -1,190 +1,121 @@
-      <div class="container-fluid">
-        <background-banner class="oppia-landing-background-image"></background-banner>
-        <div class="row oppia-landing-section-row">
-          <div class="col-sm-6 col-sm-push-6 oppia-landing-image-div">
-            <img ng-src="<[$ctrl.image1.src]>" class="oppia-landing-image" alt="<[$ctrl.image1.alt]>">
+      <div class="oppia-flex-box oppia-flex-row oppia-flex-mobile-column oppia-flex-mobile-align-center">
+        <div class="oppia-flex-item oppia-landing-page-image-container oppia-flex-mobile-order-2">
+          <img class="oppia-devices-image" ng-src="<[$ctrl.lessonInDevicesImageSrc]>" alt="Oppia lesson in different devices.">
+        </div>
+        <div class="oppia-flex-item oppia-flex-box oppia-flex-column">
+          <div class="oppia-flex-item oppia-text-align-left">
+            <h2><[$ctrl.topicTitle]> just got easier. Learn anywhere, anytime. Get started with our free and interactive lessons.</h2>
           </div>
-          <div class="col-sm-6 col-sm-pull-6">
-            <div class="oppia-landing-text-box-1">
-              <h1 class="oppia-landing-h1 oppia-text-color-green"><[$ctrl.topicTitle]> just got easier</h1>
-              <h2 class="oppia-landing-h2 oppia-text-color-green">Get your students and kids started with our free, effective <[$ctrl.subject]> lessons.</h2>
-            </div>
-            <button class="btn oppia-landing-page-button" ng-click="$ctrl.onClickGetStartedButton('teacher')">Get Started</button>
-            <button class="btn oppia-landing-page-button oppia-make-button-transparent" ng-click="$ctrl.onClickLearnMoreButton()">Learn More</button>
+          <div class="oppia-flex-item oppia-text-align-left oppia-flex-mobile-order-3">
+            <h3>Get your students and kids started with our <[$ctrl.topicTitle | lowercase]> lessons. Oppia offers story-based content for learners at different skill level. These resources are free  to you, always.</h3>
+          </div>
+          <div class="oppia-flex-item oppia-text-align-left oppia-mobile-text-align-center oppia-flex-mobile-order-2">
+            <button class="btn oppia-landing-page-button" ng-click="$ctrl.onClickGetStartedButton()">Explore <[$ctrl.topicTitle]> lessons</button>
           </div>
         </div>
       </div>
     </div>
   </div>
 
-  <div class="oppia-landing-section text-center" style="background-color: #e8e7e3">
-    <div class="oppia-landing-section-inner">
-      <div class="container-fluid">
-        <div class="row oppia-landing-section-row">
-          <div class="col-sm-6 col-sm-push-6 oppia-landing-image-div">
-            <img ng-src="<[$ctrl.image2.src]>" class="oppia-landing-image" alt="<[$ctrl.image2.alt]>">
+  <div class="oppia-landing-section text-center" style="background-color: #fff">
+    <div class="oppia-landing-section-inner oppia-flex-box oppia-flex-column">
+      <h2>What's special about our <[$ctrl.topicTitle | lowercase]> lessons?</h2>
```

Comment:
FYI, in the video, the ratios landing page shows "Fractions" here for some reason...

Reply:
I have fixed that while addressing self review comments in the last commit.